Source/Core/Core/IOS/IOS.cpp

```diff
@@ -361,7 +362,10 @@ void Kernel::AddStaticDevices()
   AddDevice(std::make_unique<Device::USB_KBD>(*this, "/dev/usb/kbd"));
   AddDevice(std::make_unique<Device::SDIOSlot0>(*this, "/dev/sdio/slot0"));
   AddDevice(std::make_unique<Device::Stub>(*this, "/dev/sdio/slot1"));
-  AddDevice(std::make_unique<Device::USB_HIDv4>(*this, "/dev/usb/hid"));
+  if ((ios_title_id & 0xFF) == 59)
```

Comment:
Technically the lower 32 bits of the title ID is used for the major version, so this would just be `GetVersion() == 59`.
In the longer term it may be worth documenting each IOS version's features and relying on feature flags instead of version numbers.